package.json

```diff
@@ -10,11 +10,11 @@
     "prepublish": "npm run build",
     "lint": "eslint --ext .js,.jsx src test stories",
     "storybook": "start-storybook -p 9001",
-    "pretest": "npm run lint",
-    "test": "npm run build && npm run test:coverage",
-    "tests-only": "npm run build && npm run test:quick",
-    "test:coverage": "babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- -R tap 'test/**/*-test.*'",
-    "test:quick": "babel-node node_modules/.bin/_mocha -R tap test/*-test.*",
+    "pretest": "npm run build && npm run lint",
+    "test": "nyc -e .jsx --reporter=lcov --reporter=text npm run test:unit",
```

Comment:
Kind of.`test:unit` itself runs `test:_unit` two times - the first time with SKIP_DOM=1 (server-side run) and then the second time with `--require ./test/.dom.js` and no `SKIP_DOM` (client-side run with dom)